src/index.js

```diff
@@ -2,18 +2,61 @@ import "./styles/normalize.css";
 import "./styles/styles.css";
 import { sortArrayByName } from './utils';
 import { moviesData } from './movies';
-import { getRoot, getMovieWrap } from './accessors';
-import { renderWelcomeLetter, renderMovies } from './render';
-
+import { getMovieWrap, getMovieContainerDiv, getMovieDetailsContainerDiv } from './accessors';
+import { renderMovies, renderMovieDetails, clearMovies } from './render';
 
 function init() {
-    const root = getRoot();
-    renderWelcomeLetter(root);
-    console.log(moviesData);
 
     const renderedMovies = renderMovies(moviesData);
+    const movieContainerDiv = getMovieContainerDiv();
+    movieContainerDiv.append(renderedMovies);
+
+
     const movieWrap = getMovieWrap();
-    movieWrap.append(renderedMovies);
+    movieContainerDiv.addEventListener('click', (evt) => {
+        let className = evt.target.getAttribute('class');
+        if (className === 'movie__card') {
+            const movie_id = evt.target.getAttribute('id');
+            const movieDetailsContainer = getMovieDetailsContainerDiv();
+            const renderedDetails = renderMovieDetails(moviesData[movie_id]);
+            movieDetailsContainer.append(renderedDetails);
+            clearMovies(movieWrap);
+        }
+
+    }, false);
+
+    const latest_mouseover_obj = {
+        class: '',
+        id: ''
+    };
+
+    movieContainerDiv.addEventListener('mouseover', (evt) => {
```

Comment:
If you're navigating between two layouts (film grid & grid details),
try to think about removing the event listener, when you change the layout